app/views/agency_update_requests/edit.html.erb

```diff
@@ -100,8 +100,7 @@
               </div>
             </div>
             <div class="panel-inline-body">
-              <%= link_to (t'buttons_t.back'), agency_update_requests_path, class: 'btn btn-primary' %>
-              <%= f.button (t'buttons_t.save'), class: "btn btn-info" %>
+              <%= link_to (t'buttons_t.back'), agency_update_requests_path, class: 'btn btn-info' %>
```

Comment:
There should still be a "Save" button when an admin wants to save the submission but not approve it or reject it yet.

Reply:
Oh, that makes sense. Thanks @micronix !